frontend/packages/ceph-storage-plugin/src/odf-external-providers/types.ts

```diff
@@ -0,0 +1,10 @@
+export type ExternalProvider = {
```

Comment:
This file and folder is to allow external providers to add their components e.g flash system.
I am keeping it separate so that I can add more details to it and maintain separate components for the external system.